src/components/BasicHead.vue

```diff
@@ -233,6 +295,11 @@ export default {
   },
   
   methods: {
+
+    showTags() {
+      this.$store.dispatch('adjust_trigger_bookmark')
```

Comment:
we need to start prefixing our vuex entries with `vxg_` to namespace them so that we don't break downstream apps
this is a big refactor, so not for now
but can start by doing this for any new names

Reply:
Roger that!